runtime/src/main/kotlin/kotlin/text/StringBuilder.kt

```diff
@@ -292,4 +642,299 @@ actual class StringBuilder private constructor (
+ *
+ * @throws IndexOutOfBoundsException or [IllegalArgumentException] when [startIndex] or [endIndex] is out of range of the [value] character sequence indices or when `startIndex > endIndex`.
+ */
+@SinceKotlin("1.3")
+@ExperimentalStdlibApi
+@Suppress("EXTENSION_SHADOWED_BY_MEMBER", "NOTHING_TO_INLINE")
+public actual inline fun StringBuilder.appendRange(value: CharSequence?, startIndex: Int, endIndex: Int): StringBuilder =
+        this.appendRange(value, startIndex, endIndex)
+
+/**
+ * Inserts characters in a subarray of the specified character array [value] into this string builder at the specified [index] and returns this instance.
+ *
+ * The inserted characters go in same order as in the [value] array, starting at [index].
+ *
+ * @param index the position in this string builder to insert at.
+ * @param value the array from which characters are inserted.
+ * @param startIndex the beginning (inclusive) of the subarray to insert.
+ * @param endIndex the end (exclusive) of the subarray to insert.
+ *
+ * @throws IndexOutOfBoundsException or [IllegalArgumentException] when [startIndex] or [endIndex] is out of range of the [value] array indices or when `startIndex > endIndex`.
+ * @throws IndexOutOfBoundsException if [index] is less than zero or greater than the length of this string builder.
+ */
+@SinceKotlin("1.3")
+@ExperimentalStdlibApi
+@Suppress("EXTENSION_SHADOWED_BY_MEMBER", "NOTHING_TO_INLINE")
+public actual inline fun StringBuilder.insertRange(index: Int, value: CharArray, startIndex: Int, endIndex: Int): StringBuilder =
+        this.insertRange(index, value, startIndex, endIndex)
+
+/**
+ * Inserts characters in a subsequence of the specified character sequence [value] into this string builder at the specified [index] and returns this instance.
+ *
+ * The inserted characters go in the same order as in the [value] character sequence, starting at [index].
+ *
+ * @param index the position in this string builder to insert at.
+ * @param value the character sequence from which a subsequence is inserted. If [value] is `null`,
+ *  then characters will be inserted as if [value] contained the four characters `"null"`.
+ * @param startIndex the beginning (inclusive) of the subsequence to insert.
+ * @param endIndex the end (exclusive) of the subsequence to insert.
+ *
+ * @throws IndexOutOfBoundsException or [IllegalArgumentException] when [startIndex] or [endIndex] is out of range of the [value] character sequence indices or when `startIndex > endIndex`.
+ * @throws IndexOutOfBoundsException if [index] is less than zero or greater than the length of this string builder.
+ */
+@SinceKotlin("1.3")
+@ExperimentalStdlibApi
+@Suppress("EXTENSION_SHADOWED_BY_MEMBER", "NOTHING_TO_INLINE")
+public actual inline fun StringBuilder.insertRange(index: Int, value: CharSequence?, startIndex: Int, endIndex: Int): StringBuilder =
+        this.insertRange(index, value, startIndex, endIndex)
+
+// Method arguments renamings
```

Comment:
```suggestion
// Method parameters renamings
```
Also they should be deprecated with replacement.